src/tests/ftest/cart/util/cart_logparse.py

```diff
@@ -1,5 +1,6 @@
 # /*
-#  * (C) Copyright 2016-2023 Intel Corporation.
+#  * (C) Copyright 2016-2024 Intel Corporation.
```

Comment:
Intel copyright should not be updated. You'll have to manually revert to 2016-2023